### ISSUE TITLE
Unable to actually begin a quiz.

### ISSUE THREAD

**Issue by Khorne101** [instructure/canvas-lms#2332]
### Summary:

Cannot begin a quiz on Firefox and Chromium while using Arch-Linux and on Chrome while using Windows

### Steps to reproduce:

 1.Have an assignment with a quiz that needs to be opened in a new window
 2.open that in a new window
 3.click begin

### Expected behavior:

The test to begin so I am able to complete it

### Actual behavior:

The begin button does absolutely nothing.

### Additional notes:

This started on March 7th, it is currently March 19th and I am still unable to complete this test on my device.

**Comment by mediumPuppy** [comment]
This seems more like an issue with the settings or content of a quiz? Are you able to contact someone able to see settings for that? At least for windows.

**Comment by Khorne101** [comment]
what do you mean by "settings" of the quiz, you mean on the course instructors end? Also the settings for the quiz should not be different for windows or linux because this is a web app, not some desktop app.

**Comment by Khorne101** [comment]
Yeah, it’s not working for me on any OS, I will talk to that classes
instructor and figure it out from there.

**Comment by Taimoor2500** [comment]
have you tried on any other environment other than os? with same configurations
